modules.d/90dmsquash-live-ntfs/module-setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+command -v
+
+check() {
+    require_binaries ntfs-3g || return 1
+    return 0
```

Comment:
This should return 255, otherwise this module gets pulled by default, and it will pull in dmsquash-live, which is probably not what you want on a normal system.